apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -389,8 +389,10 @@ const SwapForm = () => {
         `/platform/${getProviderName(exchangeRate.provider).toLowerCase()}`;
       history.push({
         pathname: providerURL,
-        params: {
+        state: {
           returnTo: "/swap",
+        },
+        params: {
```

Comment:
did not we fix this 2 weeks ago?

Reply:
@sarneijim The reason I've changed this is because in typescript, this is invalid:

```
const { push } = useHistory();
push({pathname: '/foo', params: {} })
```

There is no `params` field according to the types.

So it should be:

```
const { push } = useHistory();
push({pathname: '/foo', state: {} })
```